bundleserver/src/main/java/net/discdd/server/BundleServerApplication.java

```diff
@@ -54,35 +54,41 @@ protected void refresh(ConfigurableApplicationContext applicationContext) {
                 super.refresh(applicationContext);
             }
         }
+
         var app = new MySpringApplication(BundleServerApplication.class);
-        if (args.length > 0) {
-            Resource resource = new FileSystemResource(args[0]);
-            if (resource.exists()) {
-                try {
-                    Properties properties = PropertiesLoaderUtils.loadProperties(resource);
-                    app.setDefaultProperties(properties);
-                    args = Arrays.copyOfRange(args, 1, args.length);
-                } catch (Exception e) {
-                    logger.log(SEVERE, "Please enter valid properties file path!");
-                    System.exit(1);
-                }
-            } else {
-                logger.log(SEVERE, "Entered properties file path does not exist!");
-                System.exit(1);
-            }
-        } else {
+
+        if (args.length == 0) {
             logger.log(SEVERE, "Please enter properties file path as argument!");
             System.exit(1);
         }
 
+        Resource resource = new FileSystemResource(args[0]);
+        if (!resource.exists()) {
+            logger.log(SEVERE, String.format("Entered properties file path %s does not exist!", args[0]));
+            System.exit(1);
+        }
+
+        try {
+            var properties = PropertiesLoaderUtils.loadProperties(resource);
+            app.setDefaultProperties(properties);
+            args = Arrays.copyOfRange(args, 1, args.length);
+        } catch (Exception e) {
+            logger.log(SEVERE, "Please enter valid properties file path!");
+            System.exit(1);
+        }
+
         app.setBanner((e, s, o) -> o.println(BananaUtils.bananansi("DDD Bundle Server", Font.ANSI_SHADOW, Ansi.GREEN)));
 
         if (CommandProcessor.checkForCommand(args)) {
             // we are doing a CLI command, so don't start up like a server
             app.setWebApplicationType(WebApplicationType.NONE);
             app.setBannerMode(Banner.Mode.OFF);
             app.setLogStartupInfo(false);
+            System.setProperty("logging.level.root", "WARN");
+            System.setProperty("grpc.server.port", "-1");
+            System.setProperty("logging.pattern.console", "%-1level %msg%n");
```

Comment:
What is this property used for?

Reply:
that is to remove the timestamp and component field from the log. it makes it more readable on the screen.